app/Notifications/NewConference.php

```diff
@@ -3,9 +3,9 @@
 namespace App\Notifications;
 
 use Carbon\Carbon;
-use Illuminate\Notifications\Messages\SlackAttachment;
-use Illuminate\Notifications\Messages\SlackMessage;
 use Illuminate\Notifications\Notification;
+use Illuminate\Notifications\Messages\SlackMessage;
```

Comment:
alpha sort please

Reply:
I hope I changed all my changes back. Didn't realize the alpha sort :-)